shared/src/question-generators/recursion/formula.ts

```diff
@@ -17,27 +17,29 @@ const translations: Translations = {
     basecase: "The base case is",
     text1: "Consider the following recursive procedure `{{0}}` with integer input ${{1}}$:",
     text2: "Let ${{0}}$ be the number of stars (`*`) that the procedure prints.",
-    "long-name": "Recurrence Relation",
+    longName: "Recurrence Relation",
     question: "What is the recurrence relation of",
     name: "Recurrence",
     description: "Determine the recurrence relation of a recursive function",
     bottomnote: "Note: This field expects a string of the form `{{0}}` as input.",
-    "feedback.incomplete": "Incomplete or too complex",
```

Comment:
I assume the refactoring is to get rid of the quotes?
Not necessary in my opinion. These language objects can get quite messy and I found it helpful to group some texts using the dot names. If this is another linter issue, we should find better linter settings.

Reply:
I think the `.` has been the problem and than I also removed the quotes. But the quotes don't create a error, so I think we can keep it like: `"longName"` 👍

Reply:
Yes adding the dot to separate two words inside quotes results in an error, so I removed the dot. So at least we need to remove the dot, only the quotes can stay (they don't produce any problem).